### PR TITLE
Auto Query Fixer: implement Tokenize and some utils functions in ZetaSQL Helper server

### PR DESCRIPTION
**Pull request by mingen-pan**
Implement Tokenize and Token class
RPC call for Tokenize is set up.
Util function are developed for future features
Test case are included.

### CODE REVIEW

File: tools/zetasql_helper/zetasql_helper/local_service/BUILD [hunk 1]
```diff
@@ -24,6 +25,19 @@ cc_grpc_library(
     deps = [":local_service_cc_proto"],
 )
 
+cc_library(
+    name = "local_service",
+    srcs = ["local_service.cc"],
+    hdrs = ["local_service.h"],
+    copts = ["-Wno-sign-compare"],
```
Comment (kikkyo) [inline tools/zetasql_helper/zetasql_helper/local_service/BUILD:32]: curious what is this used for?

File: tools/zetasql_helper/zetasql_helper/local_service/local_service.proto [hunk 2]
```diff
@@ -11,6 +28,9 @@ service ZetaSqlHelperLocalService {
   rpc Hello(HelloRequest) returns (HelloResponse) {
   }
 
+  rpc Tokenize(TokenizeRequest) returns (TokenizeResponse) {
+  }
+
 }
 
 message HelloRequest {
```
Comment (kikkyo) [inline tools/zetasql_helper/zetasql_helper/local_service/local_service.proto:36]: we don't need this HelloRequest/Response right?

File: tools/zetasql_helper/zetasql_helper/local_service/BUILD (hunk 1 above)
Comment (mingen-pan) [reply tools/zetasql_helper/zetasql_helper/local_service/BUILD:32]: I took this compiler options from the ZetaSQL BUILD file. I guess it asks the compiler not to care about the sign of data types.

File: tools/zetasql_helper/zetasql_helper/local_service/local_service.proto (hunk 2 above)
Comment (mingen-pan) [reply tools/zetasql_helper/zetasql_helper/local_service/local_service.proto:36]: No we don't need it. I will remove it.

**Pull request merged**